ec2/sc-ec2-linux-jumpcloud-notebook-development.yaml

```diff
@@ -339,7 +339,8 @@ Resources:
                 }
                 DEPARTMENT=$(extract_tag_value Department)
                 PROJECT=$(extract_tag_value Project)
-                OWNER_EMAIL=$(extract_tag_value OwnerEmail)
+                PROVISIONING_PRINCIPAL_ARN=$(extract_tag_value 'aws:servicecatalog:provisioningPrincipalArn')
+                OWNER_EMAIL=${!PROVISIONING_PRINCIPAL_ARN##*/}
```

Comment:
Yes, I know I'm repeating this fix in a bunch of places. I want to try to unblock @ahayden for now and then I will backtrack and do SC-18, or SC-78 and SC-79.